### PR TITLE
Fix thow 'Unable to create .git/index.lock'

### PR DESCRIPTION
**Pull request by muwoo**
## Description

`git commit` should used after `git add .`

Issue: #64 

## Tests

- [x] All tests passed.

### CODE REVIEW

**Comment by muwoo** [comment]
Help Wanted.

I have no idea about writing the test.

File: src/gitmoji.js [hunk 1]
```diff
@@ -149,13 +149,17 @@ class GitmojiCli {
 
     if (config.getAutoAdd()) {
       execa.stdout('git', ['add', '.'])
-        .then((res) => console.log(chalk.blue(res)))
-        .catch((err) => this._errorMessage(err.stderr))
+        .then((res) => {
+          console.log(chalk.blue(res))
```
Comment (carloscuesta) [inline src/gitmoji.js]: ⚠️ Remove that console.log

File: src/gitmoji.js [hunk 2]
```diff
@@ -149,13 +149,17 @@ class GitmojiCli {
...
+          return execa.shell(commit)
```
Comment (carloscuesta) [inline src/gitmoji.js]: You can return this directly from the arrow function

Comment (muwoo) [reply src/gitmoji.js]: Thanks a lot! I've changed it.

**Pull request merged**